builtin/logical/postgresql/backend.go

```diff
@@ -35,6 +35,8 @@ func Backend() *framework.Backend {
 		Secrets: []*framework.Secret{
 			secretCreds(&b),
 		},
+
+		Clean: b.ResetDB,
```

Comment:
I don't see the actual implementation of ResetDB in the diff...did you mean to include it?